src/main/cpp/tiv.cpp

```diff
@@ -401,8 +401,11 @@ enum Mode {AUTO, THUMBNAILS, FULL_SIZE};
 
 int main(int argc, char* argv[]) {
   struct winsize w;
+  ioctl(STDOUT_FILENO, TIOCGWINSZ, &w);
+
+  if(w.ws_col == 0 && w.ws_col == 0)
```

Comment:
Cool, thanks a lot!

Can you add curlies around the conditional statement to make it more obvious? O:)

Reply:
```suggestion
  if(w.ws_col == 0 && w.ws_col == 0)
```
Sorry,  I did not understand "add curlies around the conditional"

Reply:
done